stroyprombeton/tests/tests_views.py

```diff
@@ -741,13 +741,35 @@ class ProductPrice(TestCase):
 
     fixtures = ['dump.json']
 
-    def test_price_list(self):
-        """Context for pdf generation should include Category and Products."""
-        self.response = self.client.get('/gbi/categories/1/pdf/')
+    def setUp(self):
+        self.category = models.CategoryPage.objects.filter(level=0).first()
+        self.response = self.client.get(
+            reverse('product_pdf', args=(self.category.id,))
+        )
+        self.context = self.response.context
+
+    def test_content_type(self):
+        self.assertEqual(self.response['Content-Type'], 'application/pdf')
+
+    def test_category_name(self):
+        self.assertEqual(self.context['category'].name, self.category.name)
+
+    def test_products(self):
+        self.assertIsInstance(self.context['products'], models.OptionQuerySet)
+        self.assertGreater(self.context['products'].count(), 100)
+
+    def test_tags_group(self):
```

Comment:
```suggestion
    def test_tags_are_grouped(self):
```

And we can get rid of additional comment